basetest.pm

```diff
@@ -414,6 +414,14 @@ sub next_resultname {
     }
 }
 
+sub write_resultfile {
+    my ($self, $filename, $output) = @_;
+
+    open my $fh, '>', bmwqemu::result_dir() . "/$filename";
+    print $fh $output;
+    close $fh;
```

Comment:
This would look a bit nicer with `Mojo::File`. (At first i wanted to complain about the lack of exception handling here... but then noticed the `autodie` at the top of the file 😃)
```perl
use Mojo::File 'path';
...
path(bmwqemu::result_dir(), $filename)->spurt($output);
```

Reply:
os-autoinst depends on Mojo anyways so I'll change that.